docs/local-development.md

```diff
@@ -556,6 +557,19 @@ When we started the Debugger, we were not sure if we would keep webpack and the
 
 Now that we *have* decided that working in github with webpack has a lot of benefits we could switch to JSX. We are open to switching if someone could help us do it, join the [discussion here](https://github.com/devtools-html/debugger.html/issues/1747).
 
+### Errors
+
+#### Pulling
+
+If you're running into errors associated with updating your files locally, try:
+
+2. `git checkout .`
```

Comment:
how about reset your branch to master:
1. `git fetch origin`
2. `git checkout master`
3. `git reset --hard origin/master`

Then update your node modules: `yarn nom`
Then restart the local server `yarn start`